packages/blockchain-link/package.json

```diff
@@ -47,6 +47,7 @@
         "worker-loader": "^3.0.8"
     },
     "dependencies": {
+        "@trezor/utils": "*",
```

Comment:
use real version here, blockchain-link is published to npm registry

Reply:
done, by adding real version `1.0.0`